src/advanced/bitcoin-anchoring.md

```diff
@@ -509,7 +509,9 @@ Requires [observer interval](#observer-interval) to be set.
 
 Returns the content of the anchoring transaction which anchors the
 specific block. If the selected block has not been anchored yet or if the
```

Comment:
I'd suggest to reformat as follows to increase readability:

[...] Returns `null` on any of the following conditions:

- The selected block has not been anchored yet
- The observer interval is not set
- The number of confirmations of anchoring transaction is less than the UTXO confirmations parameter